core.go

```diff
@@ -407,6 +408,17 @@ func (c *Core) RemoveLookup(lookup, lookupID, key string) (err error) {
 	return
 }
 
+// Dump will write the entire underlying db to a writer
+// NOTE: Eventually, dbl.Dump will implement it's own writer that is agnostic of database type
+func (c *Core) Dump(w io.Writer) (n int64, err error) {
+	err = c.db.View(func(txn *bolt.Tx) (err error) {
+		n, err = txn.WriteTo(w)
```

Comment:
Should this also be Dump?

Reply:
Nah, bolt calls it `WriteTo`, we're going to be developing our own Dump format. Similar to pgdump for Postgres

Reply:
that's a bolt transaction, not ours :P